py2hwsw/lib/hardware/arith_logic/iob_diff/iob_diff.py

```diff
@@ -83,7 +83,7 @@ def setup(py_params_dict):
                 "connect": {
                     "clk_en_rst_s": "clk_en_rst_s",
                     "data_i": "data_i",
-                    "data_o": "data_o",
+                    "data_o": "data_i_reg",
```

Comment:
this suffix is not allowed only _i _o _io

Reply:
but maybe it is auto created by py2, I'm not sure about this feature